ddl/job_table.go

```diff
@@ -174,15 +175,24 @@ func (d *ddl) startDispatchLoop() {
 	if d.etcdCli != nil {
 		notifyDDLJobByEtcdCh = d.etcdCli.Watch(d.ctx, addingDDLJobConcurrent)
 	}
-	ticker := time.NewTicker(1 * time.Second)
+	var ticker *time.Ticker
+	if intest.InTest {
```

Comment:
Personally, I think it is ugly to introduce `inTest` into the code, should it be directly used as a variable, such as xxxInterval, and then directly change this variable for testing, rather than into the real code

Reply:
Agreed. And by the way, why sleep for 1 second? Why not some formula with some variables like `nn * TTL`

Reply:
The original code is https://github.com/pingcap/tidb/blob/release-6.4/ddl/ddl_worker.go#L182. Fine-tuning may have been done to support parallel DDL. I think it should be similar to the original logic implementation, which has some association with TTL (lease).